.github/workflows/go-update-commenter.yml

```diff
@@ -0,0 +1,65 @@
+name: "Go update commenter"
+
+on:
+  pull_request:
+    # Only run on PR label events (in particular not on every commit)
+    types: [ labeled ]
```

Comment:
What would happen if we forgot to add the label before opening the PR? Should add the `worflow_dispatch` option to manually run this action for a particular PR?

Reply:
After discussing with platform it seems like there is no way to test a manual workflow before it is merged
(you need the workflow to be on main to be able to trigger it, even from a branch)
So I'll validate after the merge that it works properly

Reply:
As we discussed it's actually more annoying than expected, so I reverted that commit and added a card in backlog